app/models/concerns/conclusion_reviews/nbc_pdf.rb

```diff
@@ -0,0 +1,267 @@
+module ConclusionReviews::NbcPdf
+  extend ActiveSupport::Concern
+
+  def nbc_pdf organization = nil, *args
+    pdf = Prawn::Document.create_generic_pdf :portrait
+
+    put_nbc_cover_on               pdf, organization
+    put_default_watermark_on       pdf
+    put_nbc_brief_on               pdf
+    put_nbc_weaknesses_on          pdf
+    put_nbc_conclusion_on          pdf
+    put_nbc_weaknesses_detailed_on pdf
+    put_nbc_weaknesses_detected_on pdf
+
+    pdf.custom_save_as pdf_name, ConclusionReview.table_name, id
+  end
+
+  private
+
+    def put_nbc_cover_on pdf, organization
+      pdf.add_review_header organization, nil, nil
+
+      title_options = [(PDF_FONT_SIZE * 1.5).round, :center, false, :center]
+
+      pdf.move_down PDF_FONT_SIZE
+
+      width       = pdf.bounds.width
+      coordinates = [pdf.bounds.right - width, pdf.y - PDF_FONT_SIZE.pt * 14]
+      text_title  = [
+        I18n.t('conclusion_review.nbc.cover.title'),
+        review.plan_item.business_unit.name
+      ].join("\n")
+
+      pdf.bounding_box(coordinates, width: width, height: 150) do
+        pdf.text text_title, size: (PDF_FONT_SIZE * 2).round, align: :center, valign: :center, inline_format: true
+
+        pdf.stroke_bounds
+      end
+
+      pdf.move_down PDF_FONT_SIZE * 10
+
+      review_owners = review.review_user_assignments.where owner: true
+
+      responsibles = review_owners&.map do |rua|
+        rua.user.full_name
+      end
+
+      column_data = [
+        [I18n.t('conclusion_review.nbc.cover.to'), I18n.t('conclusion_review.nbc.cover.to_label')],
+        [I18n.t('conclusion_review.nbc.cover.from'), I18n.t('conclusion_review.nbc.cover.from_label')],
+        [I18n.t('conclusion_review.nbc.cover.cc'), responsibles.join("\n") ]
+      ]
+
+      width_column1 = pdf.bounds.width - PDF_FONT_SIZE * 35
+      width_column2 = pdf.bounds.width - width_column1
+
+      pdf.table(column_data, cell_style: { inline_format: true }, column_widths: [width_column1, width_column2]) do
+        row(0).style(
+          borders: [:top, :left, :right]
+        )
+        row(1).style(
+          borders: [:left, :right]
+        )
+        row(2).style(
+          borders: [:bottom, :left, :right]
+        )
+      end
+
+      pdf.move_down PDF_FONT_SIZE * 15
+      put_nbc_grid pdf
+
+      pdf.start_new_page
+    end
+
+    def put_nbc_grid pdf
+      column_data = [
+        [
+          I18n.t('conclusion_review.nbc.cover.number_review'),
+          review.identification,
+          I18n.t('conclusion_review.nbc.cover.prepared_by'),
+          I18n.t('conclusion_review.nbc.cover.internal_audit')
+        ],
+        [
+          I18n.t('conclusion_review.nbc.cover.audit_date'),
+          I18n.l(issue_date, format: '%B %Y'),'',''
```

Comment:
```suggestion
          I18n.l(issue_date, format: '%B %Y'),
          '',
          ''
```
Solo porque sino te resulta bastante raro entender que son de la misma dimensión este y el de arriba.

Reply:
Ok, gracias